backend/routes/comments.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const { check } = require("express-validator");
-const { handleValidationErrors, asyncHandler } = require("../utils");
+const { handleValidationErrors, validateComment, asyncHandler } = require("../utils");
```

Comment:
could you please remove the handleValidateErrors from the required variables, because it is not being used

Reply:
I removed the 'handleValidationErrors' from the PUT route and the require for utils. I also noticed check being required in from express-validator was unused.